scripts/generate_data.sh

```diff
@@ -2,15 +2,16 @@
 
 # Using linux date
 # For mac - brew install coreutils ; echo "alias date=gdate" >> ~/.bash_profile
-runtime="30 minutes"
+runtime="24 hours"
```

Comment:
If I were to work on it, I would put some of the value like the `runtime` or sleep time as variables 🤔 
But I don't mind as is.